modules/gcp/oslogin.go

```diff
@@ -0,0 +1,90 @@
+package gcp
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	"github.com/gruntwork-io/terratest/modules/logger"
+	"golang.org/x/oauth2/google"
+	"google.golang.org/api/compute/v1"
+	"google.golang.org/api/oslogin/v1"
+)
+
+// Add an OS Login SSH Key
+func ImportSSHKey(t *testing.T, user, key string) {
```

Comment:
Our public functions in terratest should be documented in the style:

```
ImportSSHKey will import an SSH key to GCP under the provided user identity. The `key` parameter should be the public key of the SSH key being uploaded. This will fail the test if there is an error.
```

and 


```
ImportSSHKey will import an SSH key to GCP under the provided user identity. The `key` parameter should be the public key of the SSH key being uploaded.
```